web/channels/retro_channel.ex

```diff
@@ -109,7 +109,7 @@ defmodule RemoteRetro.RetroChannel do
     {:noreply, socket}
   end
 
-  defp add_idea!(%{"body" => body, "category" => category, "userId" => user_id, "assigneeId" => assignee_id}, socket) do
+  defp add_idea!(%{"body" => body, "category" => category, "userId" => user_id, "assignee_id" => assignee_id}, socket) do
```

Comment:
why this change? not vehemently opposed to it, but it does seem inconsistent with the camelCased `userId` coming from the front end